app/uk/gov/hmrc/payetaxcalculatorfrontend/quickmodel/TaxResult.scala

```diff
@@ -38,42 +39,45 @@ object TaxResult {
       case None => throw new Exception("No answer has been provided for the question: Are you over state pension age?")
     }
 
-  def extractSalary(quickCalcAggregateInput: QuickCalcAggregateInput): BigDecimal = quickCalcAggregateInput.savedSalary match {
-    case Some(s) => s.period match {
-      case "yearly" => s.value * 100
-      case "monthly" => s.value * 100
-      case "weekly" => s.value * 100
-      case "daily" => s.value * 100
-      case "hourly" => s.value * 100
-      case _ => throw new Exception("No Salary has been provided.")
+  private[quickmodel] def extractSalary(quickCalcAggregateInput: QuickCalcAggregateInput): BigDecimal =
+    quickCalcAggregateInput.savedSalary match {
+      case Some(s) => s.period match {
+        case "yearly" => s.value * 100
+        case "monthly" => s.value * 100
+        case "weekly" => s.value * 100
+        case "daily" => s.value * 100
+        case "hourly" => s.value * 100
+        case _ => throw new Exception("No Salary has been provided.")
+      }
+      case None => throw new Exception("No Salary has been provided.")
     }
-    case None => throw new Exception("No Salary has been provided.")
-  }
 
-  def extractPayPeriod(quickCalcAggregateInput: QuickCalcAggregateInput): String = quickCalcAggregateInput.savedSalary match {
-    case Some(s) => s.period match {
-      case "yearly" => "annual"
-      case "monthly" => "monthly"
-      case "weekly" => "weekly"
+  private[quickmodel] def extractPayPeriod(quickCalcAggregateInput: QuickCalcAggregateInput): String =
+    quickCalcAggregateInput.savedSalary match {
+      case Some(s) => s.period match {
+        case "yearly" => "annual"
+        case "monthly" => "monthly"
+        case "weekly" => "weekly"
+        case _ => ""
+      }
       case _ => ""
     }
-    case _ => ""
-  }
 
 
   /**
     * This function is called "extractHours" because in "buildTaxCalc" function, the last parameter is called "hoursIn".
     * "hoursIn" does not only means hours but can also mean days.
     * buildTaxCalc will use the number returned to calculate the weekly gross pay from Daily or Hourly via those case classes.
     **/
-  def extractHours(quickCalcAggregateInput: QuickCalcAggregateInput): Int = quickCalcAggregateInput.savedSalary match {
-    case Some(s) => s.period match {
-      case "daily" => s.howManyAWeek.getOrElse(-1)
-      case "hourly" => s.howManyAWeek.getOrElse(-1)
+  private[quickmodel] def extractHours(quickCalcAggregateInput: QuickCalcAggregateInput): Int =
+    quickCalcAggregateInput.savedSalary match {
+      case Some(s) => s.period match {
+        case "daily" => s.howManyAWeek.getOrElse(-1)
```

Comment:
-1 seems to be never present during the journey flow. Maybe throwing an RuntimeException?